tcwebhooks-core/src/test/java/webhook/teamcity/settings/WebHookSettingsManagerImplTest.java

```diff
@@ -53,6 +53,10 @@ public void setup() throws JDOMException, IOException {
 		
 		projectSettings = new WebHookProjectSettings();
 		projectSettings.readFrom(ConfigLoaderUtil.getFullConfigElement(new File("src/test/resources/project-settings-test-elastic.xml")).getChild("webhooks"));
+		projectSettings.getWebHooksConfigs().forEach(c -> {
```

Comment:
There is a murky issue with old versions of ASM (which is pulled in by old versions of Jersey) when it comes to Java8 features.
I needed to convert this code to a Java 7 for loop, otherwise the code compiles with a byte code signature of Java8 and ASM can't load it.

I really need to convert to newer ASM lib since tcWebHooks 2.0.0 is now released and I no longer support TeamCity 9 (which ran on Java7). 

tcWebHooks 2.0.0 supports 2019.1 and newer only.

Reply:
See https://github.com/tcplugins/tcWebHooks/issues/156#issuecomment-623087299